src/icons/lumiAi.svg

```diff
@@ -1,3 +1,10 @@
-<svg width="10" height="20" viewBox="0 0 10 20" fill="#ADB5BD" xmlns="http://www.w3.org/2000/svg">
-<path d="M6.99568 1.38678L5.18057 7.47431L8.75114 7.47431L1.38471 19.1187L4.44849 10.3045L0.6908 10.3045L1.81651 1.38678L6.99568 1.38678Z" fill="#A7AAC6"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="23" viewBox="0 0 24 23" fill="none">
```

Comment:
The icon is not adequate 
https://github.com/lightfulhelps/ray?tab=readme-ov-file#adding-an-icon